### ISSUE TITLE
Is there a way to turn down the heat

### ISSUE THREAD

**Issue by bsgreenb** [Montana/heat-cli#1]
I turned up the heat (on Mac OS X Catalina) using `heat.sh` which worked great, but now I'm trying to turn down the heat and can't seem to reverse the operation.

Is there a work around for getting the heat low in a production environment, or is it permanently heated until you respawn the process?

Thanks in advance - Benjamin

**Comment by Montana** [comment]
Thanks for the question @bsgreenb. Yes, currently working on a subroutine that does this:

```bash
main() {
   heat_info "our informational message"
   heat_info "Our additional parameters to the program are: $@"
   heat_cool "Turning down the heat - WITH_OPT is set to $WITH_OPT (use --with-parameter)"
   subroutine1 some value "with spaces" || return 1
   random_failure || return $?
   return 0
}
subroutine1() {
   for val in "$@"; do
      log_debug "passed a subroutine a parameter: \"$val\""
   done 
   heat_info "the subroutine works"
   return 0
}
```

I would expect this in `heat-cli v1.2`. The `heat_cool` function will be one where you can toggle the `heat`. There also will be a `pressure` function.

**Comment by Cizzats** [comment]
was wondering this myself @bsgreenb, thanks @Montana!

**Comment by bsgreenb** [comment]
Excited for `heat-cli v1.2` with heat-cool functionality and more! Keep at it

Also if you could maybe write up a wiki on how developers could implement pressure in their pipelines, I know myself and a lot of others would love to read.

**Comment by Montana** [comment]
Yeah, this would be our `piping` feature. Called `piping_hots`. You can usually define this as an environment variable. Will have a wiki up within the coming weeks.

**Comment by bsgreenb** [comment]
Looking fwd to `pip install piping_hots`.  Lemme know if I could contrib with my meagre skills.

A bit of background on me: I graduated top of my class `Stanford '06`, but currently homeless. Looking for new open source opportunities to expand my network and maybe work my way up the tech ladder.

So far all I can do is `import react from 'react'`, but gotta start somewhere right?  Lemme know if you're doing any live Webinars because I'm psyched to start deploying javascript or whatever you call it.

**Comment by Montana** [comment]
There will be a `requirements.txt` file I will put out tomorrow. Some things you'll be looking out for is `Pypy` and `NumPy`. Congratulations on being the top of your class at Stanford in 2006, that must have been a lot of `pressure`. 

I think importing `React` from `react;` is one of the most important things to learn whilst becoming a programmer, second to importing `axios` from `axios;`. 

You can bet I'm working on and _improving_ `heat-cli` as we speak, and in fact - here's a sneak peek: 

<img width="929" alt="Screen Shot 2022-07-20 at 5 10 40 PM" src="https://user-images.githubusercontent.com/20936398/180103366-1c44dc63-8449-464f-9367-3122b124bad2.png">
